src/pod.rs

```diff
@@ -279,50 +279,17 @@ pub unsafe fn guarded_transmute_pod_vec<T: PodTransmutable, G: Guard>(bytes: Vec
 /// have as many instances of a type as will fit, rounded down.
 /// Extraneous data is ignored.
 #[cfg(feature = "std")]
-#[deprecated(since = "0.11.0", note = "see `pod::guarded_transmute_pod_vec()` with `PermissiveGuard` for the equivalent behavior")]
+#[deprecated(since = "0.11.0", note = "see `pod::transmute_vec()` with `PermissiveGuard` for the equivalent behavior")]
 pub unsafe fn guarded_transmute_pod_vec_permissive<T: PodTransmutable>(bytes: Vec<u8>) -> Result<Vec<T>, Error> {
-    guarded_transmute_vec::<T, PermissiveGuard>(bytes).map_err(From::from)
+    transmute_vec::<T, PermissiveGuard>(bytes).map_err(From::from)
 }
 
 /// Transform a byte vector into a vector of POD.
 ///
 /// The vector's allocated byte buffer will be reused when possible, and
 /// should not have extraneous data.
-///
```

Comment:
Any particular reason you dropped this doc?

Reply:
It's a deprecated function.

Reply:
Right, I assumed it was there for a good reason as you hadn't removed it in the PR that deprecated it.